pkg/sriov/operatorConfig.go

```diff
@@ -0,0 +1,89 @@
+package sriov
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+
+	srIovV1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OperatorConfigBuilder provides struct for SriovOperatorConfigBuilder object which contains connection to cluster and
+// SriovOperatorConfig definitions.
+type OperatorConfigBuilder struct {
```

Comment:
If you are going to update this object add Defenition as well

Reply:
structure removed as unnecessary